i18n/en/docusaurus-plugin-content-docs/version-v1.0/user/registry/zookeeper.md

```diff
@@ -3,7 +3,130 @@ title: Zookeeper Registry Center
 keywords: [Seata, Zookeeper, Registry Center]
 description: Zookeeper Registry Center.
 ---
+#  ZooKeeper Registry Center
 
-# Zookeeper Registry
+ZooKeeper serves as a critical registry center implementation within the Seata component.
 
-Welcome to claim and supplement this document.
+This document is based on Seata 1.4.2, demonstrating how to register Seata with ZooKeeper using a file as the configuration center.
+
+It is recommended to use ZooKeeper version 3.4.13 or above. The following example uses ZooKeeper version 3.4.14.
+
+## Prerequisites
+
+Before registering **Seata** with ZooKeeper, ensure that the ZooKeeper service is running. If you are unfamiliar with basic ZooKeeper usage, you can refer to the [ZooKeeper official documentation](https://zookeeper.apache.org/doc/r3.4.14/index.html).
+
+## Quick Start
+
+Integrating Seata with ZooKeeper involves simple steps, roughly categorized as "Adding Maven Dependencies" and "Configuring the Registry Center."
+
+### Server-side Registry Center Configuration
+
+Download [Seata 1.4.2 release](https://github.com/seata/seata/releases/tag/v1.4.2) and extract the files.
+
+Modify the corresponding configuration in `/conf/registry.conf`. For other configuration options, refer to the [Seata configuration reference](https://github.com/seata/seata/blob/develop/script/client/conf/registry.conf).
+
+```
+coderegistry {
```

Comment:
different versions have different configuration file types. Do not copy the same content to multiple versions.